TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Shared/PersonLayout.cshtml

```diff
@@ -0,0 +1,37 @@
+@using TeachingRecordSystem.Core.Dqt.Models;
```

Comment:
Could we move this into `Persons/Layout.cshtml`?

Reply:
Done